### PR TITLE
Devise and Flash features

### PR DESCRIPTION
**Pull request by NelsonAraujo93**
##Hello Reviewer, 🚀 
In this pull request @kensteph and i followed the next steps:

1. Install gem devise
2. Set up the project to work with devise
3. Modify sign_in view to match styles.
4. Modify sign_up view to match styles.
5. Modify change password and confirm password styles.
6. Fix linters

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (AmaduKamara) [inline app/models/user.rb]: - [x] According to the requirement, it's expected that an email confirmation feature be implemented so one is sent a confirmation link when signed up. Kindly enable comfortable so email confirmation works as expected 
- [x] Also, I tried resetting the password but I couldn't. Please work on that as well 👍🏽

```ruby
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Used devise as required 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app/models/user.rb (hunk 1 above)
Comment (NelsonAraujo93) [reply app/models/user.rb]: Thanks for your feedback

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @NelsonAraujo93, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡
 
✅ Descriptive PR 👍 
✅ Nice implementing the devise 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**